tests/DCPS/Compiler/char_literals/CMakeLists.txt

```diff
@@ -0,0 +1,40 @@
+cmake_minimum_required(VERSION 3.8...3.27)
+project(opendds_compiler_char_literals CXX)
+enable_testing()
+
+find_package(OpenDDS REQUIRED)
+include(opendds_testing)
+
+set(target_prefix "${PROJECT_NAME}_")
+set(src "${CMAKE_CURRENT_SOURCE_DIR}")
```

Comment:
```suggestion
```

You can get rid of this and where it's mentioned. Input file paths are assumed to be relative to `CMAKE_CURRENT_SOURCE_DIR` most, if not all the time. We have it in the CMake tests because they're using source files located in a very different directories from the `CMakeLists.txt`.